src/components/shared/Navigation.tsx

```diff
@@ -95,6 +95,11 @@ function Navigation() {
               </MenuButton>
               <MenuList>
                 <MenuItem>Edit Profile</MenuItem>
+                {isChapter ? (
```

Comment:
Another tip that is not important, you can use isChapter && Component instead of using this ternary operator to avoid having to explicitly state null here.